src/js/base/Context.js

```diff
@@ -54,6 +54,9 @@ export default class Context {
   }
 
   _initialize() {
+    // set default container for tooltips, popovers, and dialogs
+    this.options.container = this.options.container || this.layoutInfo.editor;
```

Comment:
`this.options.container` is only one instance.  

So it has a problem in multi summernote instances.

For example,  tooltip do not show in second tab of multi tab dialog (http://localhost:3000/examples/ondialog-multitab.html)

Reply:
This is like a origin bug.

Reply:
I made each element to have their own `options` by https://github.com/summernote/summernote/pull/3531/commits/3508d793295041e4915f4800d8b02ce3b64dc075
This also fixes the duplicated id problem when multiple summernote instances have been initiated.

Reply:
your code is good !!

Reply:
Thank you for your review @easylogic !